aikau/src/main/resources/alfresco/renderers/Selector.js

```diff
@@ -144,7 +144,8 @@ define(["dojo/_base/declare",
                   var b = lang.getObject(this.itemKey, false, this.currentItem);
                   var match = ((a || a === 0) && a === b);
                   if (match) {
-                     this.select();
+                     domClass.add(this.selectorNode, "checked");
+                     domClass.remove(this.selectorNode, "unchecked");
```

Comment:
Is this existing code? Is it easy to change to have one single "boolean class", rather than two that must not both be present?